backend/db/refresh.py

```diff
@@ -27,7 +27,7 @@
 refreshed and fetch new data from that time. Valid formats: 
 - Simple date timestamp, YYYY-MM-DD, e.g. 2022-02-22.
 - ISO 8601 datetime with timezone offset, YYYY-MM-DDTHH:MM:SS.SSSSSS+HH:MM, e.g. 2022-02-22T22:22:22.222222+00:00."""
-DEFAULT_BUFFER_HOURS = 2
+DEFAULT_BUFFER_HOURS = 0
```

Comment:
## The change
**Rationale**: We're running `fetch_failures_0_members` effectively every 20 minutes now, so this buffer is now redundant. The buffer was basically functioning to do the same thing; to find members that were not yet available when the cset was initially fetched.